src/liballoc/vec_deque.rs

```diff
@@ -1754,7 +1755,7 @@ impl<T> VecDeque<T> {
     fn grow_if_necessary(&mut self) {
         if self.is_full() {
```

Comment:
This seems like the right place to use `unlikely`, if anywhere (and not inside the is_full method)

Reply:
@bluss indeed, see #45440.